app/assets/stylesheets/base/variables.scss

```diff
@@ -4,17 +4,26 @@
 Typically, if a color is used more than once or twice,
 it should be added here as a variable. Changing the value
 of these variables will affect the colors sitewide. */
-$font-stack-body: "proxima-nova", verdana, sans-serif;
-$font-stack-heading: "futura-pt", verdana, sans-serif;
-$cu-red: #98002e;
-$cu-red-rgb-transparent: rgba(168,36,57,0.95);
-// $cu-red2: #830c28; // Old red replaced by $cu-red to keep colors standard
-// $cu-red3: #98002E; // Old red replaced by $cu-red to keep colors standard
+$futura-regular: "futura-pt", verdana, sans-serif;
+$futura-condensed: "futura-pt-condensed", verdana, sans-serif;
+$proxima-nova: "proxima-nova", verdana, sans-serif;
+$arial: Arial, verdana, sans-serif;
+
+$font-stack-body: $proxima-nova;
+$font-stack-body-new: $arial;
+$font-stack-heading: $futura-regular;
+$font-stack-heading-new: $futura-condensed;
+
+$cu-red: #a50034;
+$cu-red-rgb-transparent: rgba($cu-red, 0.95);
 $cu-dark-red: #851c2b;
 $cu-off-white: #f7f6f0;
 $cu-light-grey: #eaeaea;
 $cu-warm-light-grey: #eaeaea;
 
+$red-button-hover: #C8003F;
```

Comment:
I don't see these 2 variables `$red-button-hover` and `$red-button-active` actually being used anywhere..? Please remove

Reply:
I did a quick search in files regarding the app/assets/stylesheets directory; I did not find any instance(s) of the aforementioned two variables. I will remove.

Reply:
I'm mistaken. These two variables are used in defaults.scss. See the #theme style declaration.